### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by alexsaveliev**
- using `bash script` instead of `script`, `python file.py` instead of `file.py`
- forcing slashes as file path separator to produce the same results on Windows and Unix
- taking into account different virtualenv directory layout on Windows and Unix
- Retrieving executable file location from os.Args instead of calling srclib's toolchain.Lookup
- bundling virtualenv 13.1.0 instead of downloading it at request. Using virtualenv in program mode to create temporary environment

### CODE REVIEW

**Comment by MaikuMori** [comment]
Hey thanks for the pull requests.

What's the reason for including virtualenv instead of downloading it? That seems like a separate pull request.

I like the addition of `--isolated` and disabled version check.

Apart from the few style issues, it looks good. (I didn't test it on Windows, just reading the code).

**Comment by alexsaveliev** [comment]
> What's the reason for including virtualenv instead of downloading it? That seems like a separate pull request.

You are right, these changes weren't needed. I reverted them, now virtualenv is downloaded and installed during `make install`

**Comment by beyang** [comment]
This fails the test on Travis (https://travis-ci.org/sourcegraph/srclib-python/builds/82000040), because it fails to find the `pip` command. Not sure what changed in this diff to break that, but you can manually install pip in the `.travis.yml` file as a step in the `before-install` stage.

**Comment by alexsaveliev** [comment]
> This fails the test on Travis

Sorry, my bad. Fixed now

File: grapher/file_grapher.py [hunk 1]
```diff
@@ -309,13 +309,20 @@ def _abs_module_path_to_relative_module_path(self, module_path):
...
         pi1 = pi2 = -1
+        prev_component = None
+        prev_index = -1
         for i, component in enumerate(components):
             if component in ['site-packages', 'dist-packages']:
                 pi1 = i
                 break
             # Fallback.
-            if pi2 == -1 and component.startswith('python'):
+			# Windows case .env/lib/file.py, Unix case .env/lib/python.../file.py
```
Comment (beyang) [inline grapher/file_grapher.py]: indent this at the same level as the lines around it.

Comment (alexsaveliev) [inline grapher/file_grapher.py]: Fixed in alexsaveliev/srclib-python@4e0d5c83488f48bf8370bbc7670afe88f671869c

File: python/grapher.go [hunk 2]
```diff
@@ -286,7 +305,7 @@ func (c *GraphContext) inferSourceUnitFromFile(file string, reqs []*requirement)
 	return nil, fmt.Errorf("Cannot infer source unit for file %s", file)
 }
 
-func isSubPath(parent, child string) bool {
+func isSubPath(parent, child string) bool {
```
Comment (MaikuMori) [inline python/grapher.go]: You probably didn't want this there either.

Comment (alexsaveliev) [inline python/grapher.go]: fixed

**Comment by beyang** [comment]
LGTM, thanks!

**Pull request merged**